### PR TITLE
AWS v18.1.0

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/roadmap/issues/1630

### CODE REVIEW

**Review by whites11 — APPROVED** [review]
seems legit

File: aws/v18.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# :zap: Giant Swarm Release v18.1.0 for AWS :zap:
+
+ This release contains changes that address the several vulnerabilities. Additionally we ensure IAM permissions for Elastic File System (EFS) are set correctly on workload clusters.
```
Comment (T-Kukawka) [inline aws/v18.1.0/README.md:3]:
```suggestion
 This release contains changes that address several kubernetes vulnerabilities as well as improvements. Additionally we ensure IAM permissions for Elastic File System (EFS) are set correctly on workload clusters.
```

**Pull request merged**